examples_utils/benchmarks/command_utils.py

```diff
@@ -120,6 +120,33 @@ def get_benchmark_variants(benchmark_name: str, benchmark_dict: dict) -> list:
     return variants
 
 
+def remove_wandb_args(cmd: str) -> str:
+    """Remove all wandb args and their values from command strings.
+    """
+
+    new_arg_list = []
+    arg_list = cmd.split(" ")
+    
+    skip = False
+    for i in len(arg_list):
```

Comment:
shouldn't this part be range(len(arg_list))?

Reply:
yes, and it should be up to -1 too I think to avoid out of bounds? 

Actually I should add an empty string to the end of the list to let the i+1 work. 

Thanks.